ray-operator/apis/ray/v1/rayjob_types.go

```diff
@@ -73,27 +73,34 @@ const (
 
 type SubmitterConfig struct {
 	// BackoffLimit of the submitter k8s job.
+	// +optional
 	BackoffLimit *int32 `json:"backoffLimit,omitempty"`
 }
 
 // RayJobSpec defines the desired state of RayJob
 type RayJobSpec struct {
 	// ActiveDeadlineSeconds is the duration in seconds that the RayJob may be active before
 	// KubeRay actively tries to terminate the RayJob; value must be positive integer.
+	// +optional
 	ActiveDeadlineSeconds *int32 `json:"activeDeadlineSeconds,omitempty"`
 	// Specifies the number of retries before marking this job failed.
 	// Each retry creates a new RayCluster.
 	// +kubebuilder:default:=0
+	// +optional
 	BackoffLimit *int32 `json:"backoffLimit,omitempty"`
 	// RayClusterSpec is the cluster template to run the job
 	RayClusterSpec *RayClusterSpec `json:"rayClusterSpec,omitempty"`
```

Comment:
Didn't make this optional even though it has omitempty as it seems this should be required?